modules/ROOT/pages/functions/string.adoc

```diff
@@ -156,6 +156,65 @@ RETURN left('hello', 3)
 ======
 
 
+[[functions-lower]]
+== lower()
+
+`lower()` returns the given `STRING` in lowercase.
+
+*Syntax:*
+
+[source, syntax, role="noheader"]
+----
+lower(original)
+----
+
+*Returns:*
+
+|===
+
+| `STRING`
+
+|===
+
+*Arguments:*
+
+[options="header"]
+|===
+| Name | Description
+
+| `original`
```

Comment:
Is the argument name also restricted by GQL? Just thinking that we call it "input" for toLower()?
Also, should we go with the new argument description: "A string to be converted into lowercase."